src/library.js

```diff
@@ -4192,7 +4192,7 @@ LibraryManager.library = {
     var trace = _emscripten_get_callstack_js();
     var parts = trace.split('\n');
     for (var i = 0; i < parts.length; i++) {
-      var ret = Runtime.dynCall('iii', [0, arg]);
+      var ret = Runtime.dynCall_iii(func, 0, arg);
```

Comment:
This call site was missing reference to 'func' in the call, so it probably didn't work at all in the first place?